test/src/ima.spec.js

```diff
@@ -761,7 +761,7 @@ describe('Ima Plugin', function () {
     player.load();
   });
 
-  it('should sent vpaid true - linear', done => {
+  it.skip('should sent vpaid true - linear', done => {
```

Comment:
why skip it ?

Reply:
the ad tag gets 404

Reply:
can we use a local vast xml instead ?

Reply:
no. the mp4 of the ad itself failed. this is a vpaid I dont have another sample for it

Reply:
i've opened an issue to ima for that - https://groups.google.com/g/ima-sdk/c/BFJv6SH9VI8 so once it will be fixed i will back this test